src/lib/preprocessing/label.ts

```diff
@@ -22,7 +22,7 @@ export class LabelEncoder {
    * Fit label encoder
    * @param {any[]} X - Input data in array or matrix
    */
-  public fit(X: Type1DMatrix<string> = null): void {
+  public fit(X: Type1DMatrix<any> = null): void {
```

Comment:
Could you explain why we've set the matrix type to be any?

Reply:
The tests throws an error when we pass a number array in order to prevent that I have changed its type to any to accept any type of array.